### ISSUE TITLE
Unable to pull the latest `javasdk` dependency into a Gradle project

### ISSUE THREAD

**Issue by ajordens** [open-feature/java-sdk#56]
Using `implementation 'dev.openfeature:javasdk:0.1.0'` and getting:

```
   > Could not resolve dev.openfeature:javasdk:0.1.0.
     Required by:
         project :gate-package > project :gate-entitytags
      > Could not resolve dev.openfeature:javasdk:0.1.0.
         > inconsistent module metadata found. Descriptor: dev.openfeature:javasdk:0.1.0-SNAPSHOT Errors: bad version: expected='0.1.0' found='0.1.0-SNAPSHOT'
```

Going back to 0.0.2 which predates the `version.modifier` property works fine.

**Comment by beeme1mr** [comment]
Hi @ajordens, thanks for reporting this issue. We'll provide a fix as quickly as possible.

**Comment by beeme1mr** [comment]
@toddbaert, it looks like using an [empty version modifier](https://github.com/open-feature/java-sdk/blob/a38c2d232177a2f5705338fd0992fde4ca3d021a/.github/workflows/release.yml#L45) isn't behaving like we had hoped.

**Comment by toddbaert** [comment]
@beeme1mr you are correct I think that the empty version modifier is to blame here, and was likely never fully working. I don't think we want untransformed properties in the published pom.

I will open a PR to use release please as we do in the java contrib repo, which will solve this issue and also make releases more democratic.

Cc @justinabrahms

**Comment by justinabrahms** [comment]
@toddbaert I don't know what "was likely never fully working" means, but it was definitely working before our latest changes to the pom.xml

**Comment by toddbaert** [comment]
Consistent with what the reporter says, this has been an issue since 0.0.3, where we introduced the empty version modifier.

Compare the pom here: https://search.maven.org/artifact/dev.openfeature/javasdk/0.0.3/jar

With here: https://search.maven.org/artifact/dev.openfeature/javasdk/0.0.2/jar

I believe if somebody has the "version.modifier" set in their environment, it's causing the issue. But 100% sure on that one. I'll add this this dependency works fine for me, but I think the reporter is using Gradle and potentially has that prop defined.

To me it seems like the published pom shouldn't contain that expression (as before 0.0.3)

**Comment by toddbaert** [comment]
Confirmed I can reproduce this issue in a plain gradle build if I use versions 0.0.3 or 0.1.0 of the java-sdk. If I reach into my .m2 repo and edit the pom.xml to remove `${version.modifier}` from `<version>0.1.0</version>`, things work as expected.

**Comment by toddbaert** [comment]
@ajordens I've released `0.1.1` which should fix this issue (and indeed does for my test gradle) build. The POM looks sane now to me: https://repo1.maven.org/maven2/dev/openfeature/javasdk/0.1.1/javasdk-0.1.1.pom

Please confirm and close at your earliest convenience, and thanks for the report.

**Comment by ajordens** [comment]
Thanks for the quick fix! It's looking good now.
